### DISCUSSION TITLE
v5版本，如何附加ecs

### DISCUSSION THREAD

**Discussion by weiyakeji** [IrineSistiana/mosdns#631] [Q&A]
各位大佬，v5版本如何附加ecs信息进行查询，v4的版本方法显然已经无法使用了，对于附加ecs的应答无法缓存不用考虑

**Comment by hongjie2008** [comment]
遇到了一样的问题，ecs功能似乎不起作用，只能固定为某一个ip地址。

**Comment by hongjie2008** [comment]
帮5.x的wiki仔细看了一遍，好像没有ecs的plugin了。
仅找到一下的关于ecs的。仅能添加固定ip到ecs。但是我是需要在小范围内通过公网来部署mosdns. 开发老大能否考虑把这个功能加上
ecs
附加 ECS 至请求。支持 IPv4/6。
参数:  ecs [ip/mask] [ip/mask]
示例:  ecs 1.2.3.4/24 2001:dd8:1a::1/48

**Reply by cherinyy** [reply]
> [ arg ] 代表该参数是可选的。可省略。

[Sequence 插件 - mosdns-wiki-zh](https://irine-sistiana.gitbook.io/mosdns-wiki/mosdns-v5/ru-he-pei-zhi-mosdns/sequence-cha-jian#can-shu-bang-zhu-shuo-ming)

**Reply by zxbeltm** [reply]
exec: ecs

等同于 _no_ecs 设置？
